dockstore-webservice/src/main/java/io/dockstore/webservice/core/database/MyWorkflows.java

```diff
@@ -17,10 +17,11 @@ public class MyWorkflows {
     private WorkflowMode mode;
     private String gitUrl;
     private String description;
+    private boolean archived;
 
     @SuppressWarnings("checkstyle:ParameterNumber")
     public MyWorkflows(String organization, long id, SourceControl sourceControl, boolean isPublished, String workflowName,
-            String repository, WorkflowMode workflowMode, String gitUrl, String description) {
```

Comment:
Can split out to a following ticket, but this class looks an awful lot like a record 
(i.e. can much less verbose) 

https://www.digitalocean.com/community/tutorials/java-records-class

Reply:
> Can split out to a following ticket, but this class looks an awful lot like a record (i.e. can much less verbose)
> 
> https://www.digitalocean.com/community/tutorials/java-records-class

Good idea, I added a link to your comment in the follow-on ticket, let's do it there.

Reply:
Going to split it out, I think the database query change needs benchmarking, but isn't terribly controversial. 
The record change should be equivalent with no performance implication, but reduces maintenance/mental load.